### DISCUSSION TITLE
Add link to the dashboard in the traffic spike notifications

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#476] [Anything Else]
"Hey, any chance you could add a “View Live” button to quickly visit the live dashboard of the site?"

(feedback via email)

**Comment by ukutaht** [comment]
If the email address receiving the notification is the site owner, the dashboard link should be included: https://github.com/plausible/analytics/blob/master/lib/plausible_web/templates/email/spike_notification.html.eex#L11

Can you please specify if this is:
* The owner of the website and this link didn't get included (bug)
* A request to include the link for non-owners (?)

We could always send the link and if the user doesn't have access, the link will just 404.

**Reply by metmarkosaric** [reply]
ahh I see. this was the case where they have a team of people using Plausible but obviously they only have 1 account. user accounts within Plausible will definitely help fix this issue.

**Reply by ukutaht** [reply]
Yeah team accounts will definitely help as then we can determine everyone who should have access.

What do you think about including the link for everyone by default?

**Reply by metmarkosaric** [reply]
Some people would say do it as we share the login, but some would say don't do it as I send these to my clients/customers who don't have a log in. Perhaps best to prioritize implementation of user accounts which solves it for all.

**Reply by ukutaht** [reply]
Sounds good
